docs/source/index.rst

```diff
@@ -51,162 +46,177 @@ If you want to be sure that this code does not secretly depend on pyplot run ::
 which will prevent pyplot from being imported!
 
 
-showing
--------
+Globally Managed Figures
+========================
 
-The core of the API is `~.show` ::
 
-  import mpl_gui as mg
-  from matplotlib.figure import Figure
+The `mpl_gui.registry` module provides a direct analogy to the
+`matplotlib.pyplot` behavior of having a global registry of figures.  Thus, any
+figures created via the functions in `.registry` will remain alive until they
+have been cleared from the registry (and the user has dropped all other
+references).  While it can be convenient, it carries with it the risk inherent
+in any use of global state.
 
-  fig1 = Figure(label='A Label!')
+The `matplotlib.pyplot` API related to figure creation, showing, and closing is a drop-in replacement:
 
-  fig2 = Figure()
+::
 
-  mg.show([fig1, fig2])
+   import mpl_gui.registry as reg
```

Comment:
`reg` may need some consideration, as this will be the start of a "best practices".  I appreciate this is registry, but I wonder if end users will think of it that way, versus a list of GUI windows?  If the import were something like `mwindows` then the below would read `mwindows.show()` which makes more sense to me than `registry.show`.   But maybe "windows" isn't the right idea either?

Reply:
I agree!  As with #16 I think almost all of these names are up for debate.